<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Valve</class>
 <widget class="QWidget" name="Valve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Valve</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
